change/@fluentui-tokens-d60ecbe8-f77e-4874-9394-462749c5b06a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28639&commitId=c0bf7920be66a4c1f21960898ba504da95b54419&env=prod&clientType=FLUENTUI)


  <details><summary><b>Badge Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge Converged.tint - High Contrast.chromium.png|393| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
@jurokapsiar - the change breaks `<Badge appearance="danger" />`:
![image](https://github.com/microsoft/fluentui/assets/9615899/543b83fa-0299-4c66-a855-3a1b1888b79e)